src/app/loadout-builder/generated-sets/GeneratedSet.tsx

```diff
@@ -53,7 +53,7 @@ function GeneratedSet({
   upgradeSpendTier,
   lockItemEnergyType,
 }: Props) {
-  const defs = useD2Definitions();
+  const defs = useD2Definitions()!;
```

Comment:
Just a question for future, I generally avoid doing this for type safety. I take it you have no problem doing it as long as we know the defs are checked higher up in the component tree?

Reply:
Yeah, most of these pages won't even load w/o the defs.